cse/indexing/createIndex.py

```diff
@@ -0,0 +1,38 @@
+import os
+from cse.indexing.FileIndex import FileIndex
```

Comment:
Does not exist?

Reply:
Was renamed to `DocumentIndex`. I have created a list of ToDos ;)

Reply:
`DocumentIndex' doesn't exist as well (at least in my copy of the repo)

Reply:
sry typo: `DocumentMap` in `cse/lang/DocumentMap.py`
see refactoring in 01da17c